src/rail/estimation/algos/deepdisc.py

```diff
@@ -320,8 +323,11 @@ class DeepDiscPDFEstimator(CatEstimator):
         num_camera_filters=Param(int, 6, required=False, msg="The number of camera filters for the dataset used (LSST has 6)."),
     )
     # config_options.update(hdf5_groupname=SHARED_PARAMS)
-    inputs = [("input", TableHandle), ("metadata", JsonHandle)]
-    outputs = [("output", QPHandle), ("truth", TableHandle)]
+    inputs = [("model", ModelHandle),
```

Comment:
If we don't explicitly state that `("model", ModelHandle)` is an input, then everything falls apart. Truly a house of cards.